### ISSUE TITLE
 ONNX export mode is set to TrainingMode.EVAL, but operator 'instance_norm' is set to train=True

### ISSUE THREAD

**Issue by edwardnguyen1705** [JDAI-CV/fast-reid#727]
Dear @L1aoXingyu ,

Thanks for sharing a great work. I have been using this repo to train person embedding models.

When I convert to onnx, I observe a `WARNING`

```bash
$  python3 tools/deploy/onnx_export.py --config-file path/to/cfg_file

/usr/local/lib/python3.10/dist-packages/torch/onnx/symbolic_helper.py:1515: UserWarning: ONNX export mode is set to TrainingMode.EVAL, but operator 'instance_norm' is set to train=True. Exporting with train=True.
```

I also try [torch2trt](https://github.com/NVIDIA-AI-IOT/torch2trt), and I observe the same behavior. Moreover, the difference between outputs of pytorch and tensorrt are quite different on a person reid dataset. I attach the scripts to produce the results.
[torch2trt.zip](https://github.com/user-attachments/files/15881913/torch2trt.zip)
![delta_y](https://github.com/JDAI-CV/fast-reid/assets/7928673/b426f2fe-2145-4cbf-aa84-3839e5c48aa4)

I have googled and it shows that there are many persons having the same issue, I have tried some workarounds but they did not work, here is an [example](https://stackoverflow.com/questions/77486728/batchnorms-force-set-to-training-mode-on-torch-onnx-export-when-running-stats-ar).

Have encountered this issue and figured out a way to tackle this?
Thanks for your time.
